netbox/templates/extras/object_imageattachments.html

```diff
@@ -0,0 +1,35 @@
+{% extends base_template %}
+{% load i18n %}
+{% load render_table from django_tables2 %}
+{% load thumbnail %}
+
+{% block extra_controls %}
+  {% if perms.extras.add_imageattachment %}
+    <a href="{% url 'extras:imageattachment_add' %}?object_type={{ object|content_type_id }}&object_id={{ object.pk }}" class="btn btn-primary">
+      <span class="mdi mdi-plus-thick" aria-hidden="true"></span> {% trans "Attach an Image" %}
+    </a>
+  {% endif %}
+{% endblock %}
+
+{% block content %}
+  <div class="d-flex flex-wrap">
+    {% for object in image_attachments %}
+      <div class="thumbnail m-2">
+        {% thumbnail object.image "200x200" crop="center" as tn %}
+          <a href="{{ object.get_absolute_url }}" class="d-block" title="{{ object.description }}">
+            <img
+              src="{{ tn.url }}"
+              width="{{ tn.width }}"
+              height="{{ tn.height }}"
+              class="rounded"
+              alt="{{ object.name }}"
```

Comment:
Same as above about title/alt attributes. Although, here, I would argue more forcefully for description as the alt attribute since the description text is not otherwise displayed on the page.